src/main/java/com/andreamazzarella/contact_manager/ContactsRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.andreamazzarella.contact_manager;
+
+import java.util.List;
+
+public interface ContactsRepository {
+    List<Contact> allContacts();
+    List<Contact> findContact(final String searchTerm);
```

Comment:
I think strings are immutable in Java.. worth double checking, so may not need the `final` keyword here.